build/pipeline-uiautomation.yaml

```diff
@@ -30,37 +30,37 @@ stages:
         - visualstudio
     
     steps:
-    # Bootstrap the build
-    - template: template-build-and-prep-automation.yaml
```

Comment:
why is this all commented out?

Reply:
dead code. ui automation will be removed as all of those tests will run together with the PR build. file will be removed